### PR TITLE
Create post UI

### PR DESCRIPTION
**Pull request by AmeyaJain-25**
Issue #123 

I have created a new UI for create post.
Screenshot from 2021-03-18 00-46-38

I've made the UI looking good, attractive and with some descent CSS with matching color theme of the application.
The creator's name which was in an input tag was not looking good, so removed that and added the name tag for it.
Here's the form screenshot.
Screenshot from 2021-03-18 01-02-01

Also I have removed the regular upload choose a file button to the Upload Icon.
Screenshot from 2021-03-18 00-46-26

### CODE REVIEW

**Comment by sanjay270899** [comment]
@AmeyaJain-25 Nice Work!

But maybe text color in white would look better. Try some different color combinations and send the screenshots. Rest is fine.

**Comment by AmeyaJain-25** [comment]
Here are some screenshots.
Screenshot from 2021-03-19 00-04-36
Screenshot from 2021-03-19 00-01-56
Screenshot from 2021-03-18 23-59-10
Screenshot from 2021-03-18 23-56-38

**Comment by sanjay270899** [comment]
@AmeyaJain-25 `2nd` and `4th` are looking better. Choose one of them and then I will merge this PR.

**Comment by AmeyaJain-25** [comment]
I've made the second one and pushed the code.
Thanks @sanjay270899

File: client/src/components/Form/Form.js [hunk 1]
```diff
@@ -100,15 +101,28 @@ const Form = ({ currentId, setCurrentId }) => {
                 </Alert>
             }
             <div className={classes.contents}>
-            <TextField 
+            {/* <TextField
```
Comment (sanjay270899) [inline client/src/components/Form/Form.js:104]: Please remove this if it is not in use.

**Review by sanjay270899 — CHANGES_REQUESTED** [review]

**Comment by sanjay270899** [comment]
Fixes #125

**Pull request merged**